canonical-web-and-design/jaas-dashboard
# Improve login flow

## Done

- Rename `isLoggedIn` to `LogIn` for consistency
- Add basic styles to login screen
- Wrap all routes internal routes behind login gateway

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click through login flow and sanity check
